cmd/autoheal/awx_job.go

```diff
@@ -100,16 +100,69 @@ func (h *Healer) launchAWXJob(
 	templateId := template.Id()
 	templateName := template.Name()
 	launchResource := connection.JobTemplates().Id(templateId).Launch()
-	_, err := launchResource.Post().
+	response, err := launchResource.Post().
 		ExtraVars(action.ExtraVars).
 		Send()
 	if err != nil {
 		return err
 	}
 	glog.Infof(
-		"Request to launch AWX job from template '%s' has been sent",
+		"Request to launch AWX job from template '%s' has been sent, job identifier is '%v'",
 		templateName,
+		response.Job,
 	)
 	h.incrementAwxActions(action, rule.ObjectMeta.Name)
+
+	// Add the job to active jobs map for tracking
+	h.activeJobs.Store(response.Job, response.Job)
+
 	return nil
 }
+
+func (h *Healer) checkAWXJobStatus(jobID int) (bool, error) {
```

Comment:
In the rest of the code of the project when a function returns multiple values we name them:

```go
func (h *Healer) checkAWXJobStatus(jobID int) (finished bool, err error) {
   ...
}
```

Can we use the same idiom here?